views/email/email_partials/breach_alert.hbs

```diff
@@ -0,0 +1,14 @@
+    <tr>
+      <td valign="top" style="padding-top: 3px; padding-bottom: 10px; ">
+        <img src="{{ SERVER_URL }}/img/logos/{{ breachAlert.Name }}.png" width="40" style="width: 40px;" />
+      </td>
+      <td valign="top" style="vertical-align: top; padding-top:0px; padding-left: 10px; padding-right: 20px; padding-bottom: 10px;">
+        <p class="body-copy" style="padding: 0px; margin: 0px; color: #ffffff; margin: 0px; font-weight: 600; font-size: 16px;">{{ breachAlert.Title }}</p>
+        <p class="body-copy" style="padding: 0px; margin: 0px; color: #ffffff; font-weight: 600;">Breach date:<span class="body-copy" style="font-weight:300; padding-left: 5px; color: #ffffff">{{prettyDate breachAlert.BreachDate }}</span></p>
```

Comment:
I have old man eyes, but it looks like there is no space between set “Breach date:” and said date.

Reply:
Oh, maybe you’re using css padding instead of an nbsp

Reply:
Using padding here, but maybe the nbsp is better/more reliable in the uncertain and unpredictable wilds of email clients?

Reply:
👍 `&nbsp;` here!

Reply:
oops just saw this! switching and pushing again